.travis.yml

```diff
@@ -303,16 +309,17 @@ script:
  - $TRAVIS_BUILD_DIR/.travis/script-$TRAVIS_OS_NAME.sh
  - $TRAVIS_BUILD_DIR/.travis/test.sh
 # only on osx for now
- - echo $DO_DEPLOY
- - if [[ ! -z "$DO_DEPLOY" ]]; then echo hi; fi
- - if [[ -z "$DO_DEPLOY" ]]; then echo hi; fi
- - if [[ ! -z "$DO_DEPLOY" ]]; then $TRAVIS_BUILD_DIR/.travis/package-$TRAVIS_OS_NAME.sh; fi
+ - echo $DEPLOY_AWS_PREFIX
+ - echo $DEPLOY_GITHUB_SUFFIX
```

Comment:
yeah, I thought it might've been left accidentally. However, my point was that I'd personally like a little more verbosity with deployment, to help knowing what's happening. I understand the above might not be the way to go.
I was thinking about something along those lines:
```
will try to deploy / will not try to not deploy to AWS
will try to deploy / will not try to not deploy to Github
```
before the deploy step, and
```
deployed / not deployed to AWS
deployed / not deployed to Github
```
during/after, because deployment is conditional on the commit being in the SC repo, having a tag etc.

What do you think about adding these? is this too much?

Reply:
i think it's just not very helpful to see this information there. most of this is already logged and documented in other ways. for example "Skipping a deployment with the {s3,releases} provider because [reason]", which appears at the end of a branch commit build log in nice bright bold yellow text, is straightforward enough once you know that we deploy to s3 and github releases. these additional log lines would be hidden inside drop-downs in the interactive build log view, too, making them less discoverable.

btw, there are like 5 AWS services we can potentially deploy to through travis, so it's clearer to refer to it as s3 rather than AWS. the reason AWS_KEY and AWS_SECRET are named what they in our scripts is because they are credentials for an AWS account, and could be reused for other services.